ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   openshift/windows-machine-config-operator:
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
This should continue to be `true`. Setting this to `false` disables automated triggers of the job on pull requests as per the [docs](https://docs.ci.openshift.org/docs/how-tos/contributing-openshift-release/#tolerated-changes-to-generated-jobs).

Applies to all instances.

Reply:
However, given the [upstream docs](https://github.com/openshift/release/pull/19745#discussion_r659023023) you pointed us says otherwise 🤷🏾‍♂️

Reply:
Yeah.. I'll leave the changes and watch WMCO CI, as you suggested.